modules/apps/notification/notification-rest-impl/rest-openapi.yaml

```diff
@@ -1,5 +1,51 @@
 components:
     schemas:
+        NotificationQueueEntry:
+            properties:
+                actions:
+                    additionalProperties:
+                        additionalProperties:
+                            type: string
+                        type: object
+                    readOnly: true
+                    type: object
+                bcc:
+                    type: string
+                body:
+                    type: string
+                cc:
+                    type: string
+                className:
+                    type: string
+                from:
+                    type: string
+                fromName:
+                    type: string
+                id:
+                    format: int64
+                    readOnly: true
+                    type: integer
+                notificationTemplateId:
+                    format: int64
+                    readOnly: true
+                    type: integer
```

Comment:
@p-albuquerque Can you remove this property (`notificationTemplateId`) for now? Just from the DTO. Thx.